Libraries/test/TestServerlessApp.IntegrationTests/IntegrationTestContextFixture.cs

```diff
@@ -51,6 +55,64 @@ public IntegrationTestContextFixture()
             Assert.False(string.IsNullOrEmpty(RestApiUrlPrefix));
         }
 
+        private async Task DeployTestServerlessApp()
```

Comment:
let's make it a generic method like we have other repo for command executions.

Reply:
Fixed